main_ahmadsadik.py

```diff
@@ -0,0 +1,38 @@
+import random
+
+print("Hello!")
+
+while True:
+    try:
+        names = input("Please enter the names separated by a space:").split()
+
+        if len(names) == 0:
+            raise ValueError()
+
+    except:
+        pass
+    else:
+        break
+
+
+def RNG():
+    global names
+    nameslist = list(set(names))
+    while True:
+        try:
+            random_number = input("Please enter how many names you would like: ")
+
+            name = random.sample(nameslist, k=int(random_number))
+
+            if int(random_number) == 0:
+                raise ValueError()
+
+            print("The name(s) are:", name)
+
+        except:
+            pass
+        else:
+            break
+
+
+RNG()
```

Comment:
It's good practice to write the function call within the if __name__ == "__main__": syntax. Look at the article for reference: https://realpython.com/if-name-main-python/#:~:text=If%20the%20__name__,Python%20skips%20the%20indented%20code.